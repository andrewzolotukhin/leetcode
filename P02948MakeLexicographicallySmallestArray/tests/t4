10 15
5 100 44 45 16 30 14 65 83 64
