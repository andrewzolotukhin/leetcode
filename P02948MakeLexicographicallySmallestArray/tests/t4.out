5
100
14
16
30
44
45
64
83
65
